src/components/Text/Text.js

```diff
@@ -1,10 +1,18 @@
 import styled from 'styled-components';
-import { fontSmall } from '../../styles/designTokens';
+import { fontMedium, fontWeightNormal, fontWeightBold, spacingSmall } from '../../styles/designTokens';
 
-const Text = styled.span`
-  font-size: ${fontSmall};
+const Text = styled.p`
+  font-size: ${fontMedium};
```

Comment:
With such big font-size for basic texts, the App looks like prepared for the 80 years old customers :D. Imo it is too big ;)